server/utils/drupal/drupal-forums.js

```diff
@@ -106,6 +127,8 @@ function getSortParams({ sortBy, sortDirection, tid, uuid }){
     const direction = !sortDirection? 'DESC' : 'ASC';
 
     let sortQueryString = '';
+    sortQueryString += `&sort[sort-weight][path]=weight`
```

Comment:
I would reccomande to use URL or URLSearchParams api in the future
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams
https://developer.mozilla.org/en-US/docs/Web/API/URL